### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by mesudpasic**

### CODE REVIEW

File: Module.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+version: 1.0.0
+name: OPCUAReader
```
Comment (kubagrzelak) [inline Module.yaml]: please add spacebar `OPC UA Reader`

File: Module.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+description: OPC UA Reader reads values for variables and forwards the payload to next module.
+icon: https://icons-020-demo.s3.eu-central-1.amazonaws.com/mi_OPCUA_input.png
+type: process
+tags:
+  - OPC UA Reader
+  - process
+categories:
+  - id: category
+    name: Process
```
Comment (kubagrzelak) [inline Module.yaml]: Make category `Policy`

File: Module.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
+version: 1.0.0
+name: OPCUAReader
+description: OPC UA Reader reads values for variables and forwards the payload to next module.
+icon: https://icons-020-demo.s3.eu-central-1.amazonaws.com/mi_OPCUA_input.png
```
Comment (kubagrzelak) [inline Module.yaml]: Again, please ask Christina for OPC UA process icon

File: README.md [hunk 4]
```diff
@@ -1,2 +1,63 @@
...
+### Module Specific
+
+### Set by the weeve Agent on the edge-node
+
+| Environment Variables | type   | Description               |
+| --------------------- | ------ | ------------------------- |
+| MODULE_NAME           | string | Name of the module        |
+| INGRESS_HOST          | string | Host where app is running |
+| INGRESS_PORT          | string | Port where app is running |
+
```
Comment (kubagrzelak) [inline README.md:42]: Sample Input?

**Review by kubagrzelak — CHANGES_REQUESTED** [review]

File: Module.yaml (hunk 1 above)
Comment (mesudpasic) [reply Module.yaml]: done.

File: Module.yaml (hunk 2 above)
Comment (mesudpasic) [reply Module.yaml]: done.

File: Module.yaml (hunk 3 above)
Comment (mesudpasic) [reply Module.yaml]: requested.

File: README.md (hunk 4 above)
Comment (mesudpasic) [reply README.md:42]: not really, we don't have inputs for this module.

File: Module.yaml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+      - no
+  - key:  OPC_UA_USERNAME
+    description: Username for authentication
+    value: ''
+    default: ''
+    type: string
+    name: OPC UA Username
+    dependencies:
+      - key: REQUIRES_AUTHENTICATION
+        value: yes
```
Comment (kubagrzelak) [inline Module.yaml]: dependencies field is not supported in V0 (we support in V1). This module is currently in V0 so please remove it.

File: Module.yaml [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+  - key:  OPC_UA_PASSWORD
+    description: Password for authentication
+    value: ''
+    default: ''
+    type: string
+    name: OPC UA Password
+    dependencies:
+      - key: REQUIRES_AUTHENTICATION
+        value: yes
```
Comment (kubagrzelak) [inline Module.yaml]: Same

File: Module.yaml (hunk 5 above)
Comment (mesudpasic) [reply Module.yaml]: removed.

File: Module.yaml (hunk 6 above)
Comment (mesudpasic) [reply Module.yaml]: removed.

File: Module.yaml (hunk 3 above)
Comment (mesudpasic) [reply Module.yaml]: module is changed to input based on our last call.

**Review by nithinsaii — APPROVED** [review]
Module.yaml looks good and container runs fine locally.

**Pull request merged**